src/Nox.Integration/Services/NoxIntegrationContext.cs

```diff
@@ -25,16 +28,22 @@ public void Initialize()
             var instance = new NoxIntegration(_logger, integrationDefinition);
             instance.WithReceiveAdapter(integrationDefinition.Source, _solution.Infrastructure?.Dependencies?.DataConnections);
             instance.WithSendAdapter(integrationDefinition.Target, _solution.Infrastructure?.Dependencies?.DataConnections);
-            
             _integrations.Add(instance);
```

Comment:
_integrations could be a Dictionary<string,NoxIntegration> that way we would search by integration name in O(1), no need to do a O(n) every time

Reply:
Implemented